pubspec.yaml

```diff
@@ -17,16 +17,16 @@ dependencies:
   path: ^1.8.2
   win32: ^5.5.1
   cross_file: ^0.3.3+7
-  web: ^0.5.1
+  web: ^1.0.0
 
 dev_dependencies:
-  lints: ^3.0.0
+  lints: ^4.0.0
   flutter_test:
     sdk: flutter
 
 environment:
-  sdk: ">=2.19.0 <4.0.0"
-  flutter: ">=3.7.0"
+  sdk: ">=3.4.0 <4.0.0"
```

Comment:
Per the changelog for `web: 1.0.0`, Dart 3.4.0 is required. Flutter 3.22.0 is the first Flutter version that ships with that Dart SDK.

See https://github.com/dart-lang/web/blob/main/CHANGELOG.md#100